lib/widgets/banner/banner.dart

```diff
@@ -0,0 +1,148 @@
+import 'package:flex_ui/flex_ui.dart';
+import 'package:flex_ui/tokens/sizes.dart';
+import 'package:flutter/material.dart';
+import 'package:widgetbook/widgetbook.dart';
+import 'package:widgetbook_annotation/widgetbook_annotation.dart' as widgetbook;
+
+enum FlexBannerImageLayout {
+  /// Image is firstly visible, the rest of the element are shown after the
+  /// image (i.e. on the right if the orientation is horizontal, at the bottom
+  /// if orientation is vertical)
+  first,
+
+  /// Image is lastly visible, the rest of the element are shown before the
+  /// image (i.e. on the left if the orientation is horizontal, at the top
+  /// if orientation is vertical)
+  last,
+}
+
+class FlexBanner extends StatelessWidget {
+  const FlexBanner({
+    super.key,
+    required this.title,
+    required this.image,
+    this.orientation = Axis.horizontal,
+    this.imageLayout = FlexBannerImageLayout.last,
+    this.theme,
+    this.heading,
+    this.description,
+    this.button,
+  });
+
+  final String title;
```

Comment:
This as a `String` is nice for CMS purposes, but maybe optional `titleWidget` override or `style` property?

Reply:
I agree with the Widget type, String is too limited. 

I haven't seen this pattern very often (String property and optional Widget to override). Additional properties are more confusing than the String shortcut is beneficial IMO. I think I'll go with the Widget type.